miso-web/src/main/webapp/pages/editSample.jsp

```diff
@@ -868,6 +868,8 @@
       </ul>
   --%>
   <a name="library"></a>
+  
+  <c:if test="${ detailedSample and sampleCategory eq 'Aliquot' }">
```

Comment:
This should be shown for plain samples as well
(`!detailedSample or sampleCategory eq 'Aliquot'`)